platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/MerkleDbStatistics.java

```diff
@@ -240,6 +249,10 @@ public void registerMetrics(final Metrics metrics) {
                     metrics,
                     DS_PREFIX + COMPACTIONS_PREFIX + LEVEL_PREFIX + level + "_hashesSavedSpaceMb_" + label,
                     "Saved space during compactions of level %s, hashes store, %s, Mb".formatted(level, label)));
+            hashesStoreTotalSpaceMbList.add(buildDoubleAccumulator(
+                    metrics,
+                    DS_PREFIX + COMPACTIONS_PREFIX + LEVEL_PREFIX + level + "_hashesTotalSpaceMb_" + label,
```

Comment:
Since the new metrics are not specific to compactions, I'd add them to FILES category rather than to COMPACTIONS.

Reply:
Agreed